migrator.go

```diff
@@ -477,7 +477,7 @@ func (m Migrator) ColumnTypes(value interface{}) (columnTypes []gorm.ColumnType,
 			}
 
 			if column.DefaultValueValue.Valid {
-				column.DefaultValueValue.String = regexp.MustCompile(`'?(.*\b|)'?:+[\w\s]+$`).ReplaceAllString(column.DefaultValueValue.String, "$1")
+				column.DefaultValueValue.String = parseDefaultValueValue(column.DefaultValueValue.String)
```

Comment:
Should we ensure that the DefaultValueValue ​​are consistent between drivers? Now the unit test is failing.

https://github.com/go-gorm/mysql/blob/master/migrator.go#L296
https://github.com/go-gorm/sqlite/blob/master/ddlmod.go#L146

relation to https://github.com/go-gorm/gorm/pull/6579

Reply:
I don't have enough experience to make changes other drivers, so I made some changes to this one:
https://github.com/go-gorm/postgres/pull/222

I'm not sure the current approach to removing single quotes is a good decision.

Reply:
There is no impact on the API, strings.Trim will actually be executed in gorm, but relatively independent unit tests will be affected